cli/import_map.rs

```diff
@@ -53,13 +53,15 @@ impl ImportMap {
     );
 
     // Load the contents of import map
-    match fs::read_to_string(&resolved_path) {
-      Ok(json_string) => ImportMap::from_json(base_url, &json_string),
-      _ => panic!(
-        "Error retrieving import map file at \"{}\"",
-        resolved_path.to_str().unwrap()
-      ),
-    }
+    fs::read_to_string(&resolved_path)
+      .map_err(|err| {
+        ImportMapError::new(&format!(
+          "Error retrieving import map file at \"{}\": {}",
+          resolved_path.to_str().unwrap(),
+          err.to_string()
+        ))
+      })
```

Comment:
Looks good

Reply:
I think the error type should be changed to eg. `InvalidInput` - it's not an ImportMapError per se, now I think it should only be used when there's resolution error related to import map or it's structure is invalid. It's similar to how `tsconfig.json` is treated.

Reply:
@bartlomieju I thought that was the case, and even considered returning `Result<Result<Self, ImportMapError>, io::Error>>` :( In the end I decided it would be simplest to just re-imagine the meaning of `ImportMapError`, but I'm not sure. Is there a better way of handling both error types? Or is the nested `Result` alright?

Reply:
You can just change the signature of `load` to: `pub fn load(base_url: &str, file_name: &str) -> Result<Self, InvalidInput>` or `pub fn load(base_url: &str, file_name: &str) -> Result<Self, ErrBox>`. Both `ImportMapError` and `InvalidInput` can be downcast to `ErrBox` which encapsulates error handling in `//cli`

Reply:
@bartlomieju Done. Thanks 👍